portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,16 +25,25 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 import java.util.List;
+import java.time.format.DateTimeFormatter;  
+import java.time.LocalDateTime;    
 
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = getParameter(request, "comment", "");
+    String name_of_user = getParameter(request, "name", "");
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");  
+    LocalDateTime now = LocalDateTime.now();  
+    String currentTime = dtf.format(now);
```

Comment:
considering now is only used in one place, you can combine these two line into one without losing clarity.

String currentTime = dtf.format(LocalDateTime.now());